manifests/install.pp

```diff
@@ -18,7 +18,7 @@
   $python = $::python::version ? {
     'system' => 'python',
     'pypy'   => 'pypy',
-    default  => "${python::version}",
+    default  => "${python::version}", # lint:ignore:only_variable_string
```

Comment:
@cryptk My only concern is that `$python` is stringified on purpose and by changing the current behavior it will break something that is expecting that `$python` is a string.

I'm in favor of doing it your way, though that needs to be tested. Please send a PR that changes the behavior.